lib/vimgolf/cli.rb

```diff
@@ -177,8 +200,6 @@ def put(id = nil)
       VimGolf.ui.error "If the error persists, please report it to github.com/igrigorik/vimgolf"
     end
 
-    private
```

Comment:
I did nothing to the `private` statement, but I had to move most of the old `.put()` into `.play()`, which had to be below `private`. (Thor got mad at me.) Diff is confused. There's a `+ private` higher.